provisioning/vars.yml

```diff
@@ -4,14 +4,11 @@
 
 nodejs_app_name: first-discovery-server
 
-nodejs_app_commands:
-  - npm run fullinstall
-
 nodejs_app_start_script: index.js
 
 nodejs_app_tcp_port: 8088
 
-nodejs_version: 0.10.40
+nodejs_version: 0.12.7
```

Comment:
We may as well just move all the way up to our LTS version of 4.2.1 since I don't think we have any sensitive dependencies in this project.